src/transformState.js

```diff
@@ -4,8 +4,29 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (true) {
```

Comment:
Instead of using 'switch (true)', it is more appropriate to use 'switch (action.type)'. This will make the code easier to read and understand.